src/analytics/resetIntegrations.ts

```diff
@@ -0,0 +1,20 @@
+// reset segment integrations (pendo, intercom, ...) when API key has changed
+const resetIntegrations = () => {
+  console.log('Calling reset');
```

Comment:
Can we log this with `logger` instead of plain' old `console.log`?

Reply:
I tell you what. It should not even be there :smile: